bcda/testUtils/utils.go

```diff
@@ -460,3 +457,10 @@ func MakeTestServerWithInternalServerErrAuthTokenRequest() *httptest.Server {
 	})
 	return httptest.NewServer(router)
 }
+
+func ContextTransactionID() *http.Request {
+	r := httptest.NewRequest("GET", "http://bcda.cms.gov/api/v1/Group/$export", nil)
```

Comment:
Wouldnt it be better to keep the URL out of code? with parameters? that way if URL changes , you wouldnt need to change code and redeploy

Reply:
normally I would say yes - httptest requires a valid path and this func only exists as a test helper for requests that reach out to SSAS and get a token.